main/dbus/files/dbus.user

```diff
@@ -1,5 +1,7 @@
 # dbus session bus user service
 
-type               = process
-command            = /usr/libexec/dbus-session.wrapper --print-address=4
+type = process
+command = /usr/libexec/dbus-session.wrapper --print-address=4
 ready-notification = pipefd:4
+# So dbus services don't start in /etc/dinit.d/user :')
+working-dir = $HOME
```

Comment:
is this supposed to be left unquoted? i'm not sure whether dinit does the weird shell shenanigans

Reply:
it doesn't p sure